src/test/kotlin/ObserverCRMTest.kt

```diff
@@ -0,0 +1,14 @@
+import core.api.crm.controller.CrmController
+import core.holder.DynamicContextHolder.getConfig
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Test
+
+class ObserverCRMTest : BaseTest() {
+
+  @Test
+  fun `verify that value cookie AuthUser set in session context `() {
```

Comment:
that AuthUser cookie value set in SessionContext when make CRM authorisation service call